packages/doctor/README.md

```diff
@@ -2,7 +2,7 @@ appium-doctor
 ===================
 Attempts to diagnose and fix common Node, iOS and Android configuration issues before starting Appium.
 
-[![Build Status](https://travis-ci.org/appium/appium-doctor.svg?branch=master)](https://travis-ci.org/appium/appium-doctor)
+[![Build Status](https://travis-ci.com/appium/appium-doctor.svg?branch=master)](https://app.travis-ci.com/github/appium/appium-doctor)
```

Comment:
We no longer has used Travis, so this PR can be the removal

Reply:
So what do you use for build CI?

Reply:
GitHub Actions -> https://github.com/appium/appium/actions/workflows/build.yml

Reply:
And I assume there is no specific build for appium doctor. So I should refer to the appium build when im updating the build status for appium doctor.

Reply:
Yes, for current CI configuration. We may be able to configure CI for each package in this repo, but I think current execution time of entire this repo is acceptable, then we can keep simple CI tasks so far

Reply:
I've removed the build status of Travis as discussed.